docs/run.md

```diff
@@ -128,11 +128,13 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete. Marketplace VM images can be found according to the instructions [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage)
 
     ```bash
-    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>"
+    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v "marketplace_image:<marketplace image string>"
```

Comment:
Can we add the marketplace_image as a note, which is below the example?
1. This doc is a quick start, and it helps users to have a quick success experience. So its arguments should be as less as possible.
2. Rather than marketplace_image, the vhd is another important argument. LISA supports it, but we haven't make it in runbook.